Modules/Core/Common/include/itkNeighborhoodAllocator.h

```diff
@@ -63,7 +64,7 @@ class NeighborhoodAllocator
   void
   Allocate(unsigned int n)
   {
-    m_Data.reset(new TPixel[n]);
+    m_Data = make_unique_for_overwrite<TPixel[]>(n);
```

Comment:
The comments for _Allocate_ and _Deallocate_
`/** Allocates memory using new() */`
`/** Deallocates memory using delete[](). */`
are probably outdated, BTW.

Reply:
@issakomi Well... strictly speaking, those old comments are still correct, as `make_unique_for_overwrite` and `unique_ptr<T[]>` _do_ use `new` and `delete[]` (as they should). However, I think it should be considered an implementation detail of NeighborhoodAllocator. Moreover, removing them from the comments reduces the noise when searching for the remaining "naked" `new`s and `delete[]`s in the source code  😃 So thanks Mihail 👍 Please check the commit that I just added!